src/Rx/Weno/file_download.php

```diff
@@ -0,0 +1,179 @@
+<?php
```

Comment:
will this script need an access control(s) (ie. what type of users will need access to this script?).